### PR TITLE
In Praise of Buddha’s Acts

### PR DESCRIPTION
**Pull request by JJM317**

### CODE REVIEW

File: _content/monographs/in-praise-of-buddha’s-acts_willemen-charles.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: "Buddhacarita: In Praise of Buddha’s Acts"
+translator: "Charles Willemen"
+external_url: "https://bdkamerica.org/download/1896"
+drive_links:
+  - "https://drive.google.com/file/d/1ogsp86UfZguXkm1Z_qVrd2fCggIxKfX0/view?usp=sharing"
+course: rebirth-stories
```
Comment (khemarato) [inline _content/monographs/in-praise-of-buddha’s-acts_willemen-charles.md:7]: Hmmm...  this probably doesn't belong here huh... 🤔  Where does it belong though...  `canonical-poetry`? 🤔

File: _content/monographs/in-praise-of-buddha’s-acts_willemen-charles.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+tags:
```
Comment (khemarato) [inline _content/monographs/in-praise-of-buddha’s-acts_willemen-charles.md:8]: definitely should have the `buddha` tag though.  And `mahayana-roots`?

File: _content/monographs/in-praise-of-buddha’s-acts_willemen-charles.md (hunk 1 above)
Comment (JJM317) [reply _content/monographs/in-praise-of-buddha’s-acts_willemen-charles.md:7]: ya, I used canonical-poetry as a tag, but would make sense to use it as the course

File: _content/monographs/in-praise-of-buddha’s-acts_willemen-charles.md (hunk 2 above)
Comment (JJM317) [reply _content/monographs/in-praise-of-buddha’s-acts_willemen-charles.md:8]: Ya, I like mahayana-roots

**Pull request merged**